src/common/services/Database/entities/SyncLog.ts

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
```

Comment:
Very basic entity for maintaining sync logs. Sync queue is a simple adapter over the top which orders the records by timestamp.